c-api-examples/paraformer-c-api.c

```diff
@@ -0,0 +1,80 @@
+// c-api-examples/paraformer-c-api.c
```

Comment:
please follow
https://github.com/k2-fsa/sherpa-onnx/blob/9f06b059d7cbfc2137c9beeeee32acc57c8b1ca2/c-api-examples/CMakeLists.txt#L24-L31
to change CMakeLists.txt to include newly added files.